test/capable/capable.lua

```diff
@@ -9,7 +9,7 @@ package.cpath = './?.so;' .. package.cpath
 
 require('memcached').create(
     'memcached',
-    os.getenv('LISTEN'):match(':(.*)')
+    '20509'
```

Comment:
Add a comment. Now it looks like some kind of magic number.

Reply:
Added